docs/UserGuide.md

```diff
@@ -2,16 +2,47 @@
 layout: page
 title: User Guide
 ---
+* Table of Contents 
+{:toc}
+--------------------------------------------------------------------------------------------------------------------
+## 1. Introduction
+### 1.1. About StoreMando
+Greetings, and welcome to StoreMando!
+
+Looking for an all-in-one solution for your inventory management needs? Look no further!
+
+
+StoreMando is an integrated platform fully customised for residents in households, residential colleges and halls, with 
+the aim of helping you manage your items effectively and efficiently. StoreMando allows you to keep track of your items' 
+whereabouts, quantities and expiry dates with a few simple commands. Furthermore, StoreMando also comes with an inbuilt 
+reminder feature to help you remember items that are expiring. It looks like you will never have to worry about consuming 
```

Comment:
maybe change from 'remember items' to 'keep track of items' :)

Reply:
Alright, fixed!